app/i18n/docs/en/InfoModals/SendFromUnmixed.md

```diff
@@ -0,0 +1,3 @@
+## Allow Send from other unmixed accounts
```

Comment:
I think we can remove this heading, because the modal already has one. This one isn't rendered properly anyway, it just appears as a bold line at the start of the paragraph.